ddev/tests/helpers/hatch.py

```diff
@@ -0,0 +1,15 @@
+# Generic hatch environment configuration
```

Comment:
This has been added in helpers because it is going to be needed in the future anytime we need to mock the calls to hatch, that way we have it centralized. Will probably add other helper methods here when working on the fix for the weekly latest job.